### DISCUSSION TITLE
Swipe to review?

### DISCUSSION THREAD

**Discussion by OliPassey** [Radiokot/photoprism-android-client#74] [Ideas]
Would love it if the app could provide a swipe left / right interface to approve / deny photos in the review list. 

I have thousands of pics in review and this would help eventually get the list down to zero. 

Possibly a niche request 🤷

**Comment by Radiokot** [comment]
Hi. That's really an interesting concept of reviewing photos 😄 

Did you know you can review photos in a quite common way in the web app by switching view to the card mode?

<img width=400 src="https://github.com/Radiokot/photoprism-android-client/assets/5675681/8b2c124b-155e-4897-9947-d818f130a642" />

- If you press ❌, the photo gets archived
- If you press ✔, the photo gets approved (going to be shown in the search results)
- Whatever you choose, the card gets dismissed and the next one moves up

**Reply by OliPassey** [reply]
I didn't! Thanks for sharing will take a look.
